CsprSdkStandardTestsNet/Test/Features/deploys.feature

```diff
@@ -1,11 +1,30 @@
 Feature: Deploys
 
-    Scenario: Put Transfer Deploy
+    Scenario: Transfer Deploy
```

Comment:
If you change this please also updates happen in https://github.com/casper-network/sdk-std-test-resources